### PR TITLE
Update SDKs and demo apps to 2.3.0

### PR DESCRIPTION
**Pull request by Cloudwu1**
Things to consider before you submit the PR:

* [X] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [X] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

### CODE REVIEW

File: Apps/Contoso.UWP.Demo/MainPage.xaml [hunk 1]
```diff
@@ -2,14 +2,16 @@
     x:Class="Contoso.UWP.Demo.MainPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:Contoso.UWP.Demo"
+    xmlns:local="using:Contoso.Demo.Puppet"
```
Comment (MatkovIvan) [inline Apps/Contoso.UWP.Demo/MainPage.xaml]: Does it work after this change?

File: Apps/Contoso.WPF.Demo.DotNetCore/Property.cs [hunk 2]
```diff
@@ -1,5 +1,8 @@
-﻿using System.ComponentModel;
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
 using System.Runtime.CompilerServices;
+using System.Text;
```
Comment (MatkovIvan) [inline Apps/Contoso.WPF.Demo.DotNetCore/Property.cs]: This usings look unused

File: Apps/Contoso.UWP.Demo/App.xaml.cs [hunk 3]
```diff
@@ -82,6 +117,7 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
             throw new Exception("Failed to load Page " + e.SourcePageType.FullName);
         }
 
+
```
Comment (MatkovIvan) [inline Apps/Contoso.UWP.Demo/App.xaml.cs]:
```suggestion
```

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/App.xaml [hunk 4]
```diff
@@ -1,9 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Application xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              x:Class="Contoso.Forms.Demo.App">
     <Application.Resources>
         <!-- Application resource dictionary -->
-        <Color x:Key="SwitchCellOnColor">Accent</Color>
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/App.xaml]: As far as I remember it was some bugfix, please revert or make sure that isn't required anymore

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/AddPropertyContentPage.xaml.cs [hunk 5]
```diff
@@ -17,14 +17,15 @@ public AddPropertyContentPage()
...
+            Property addedProperty = new Property(NameEntry?.Text, ValueEntry?.Text);
+            PropertyAdded?.Invoke(addedProperty);
             await Navigation.PopModalAsync();
         }
 
         async void Cancel(object sender, EventArgs e)
         {
             await Navigation.PopModalAsync();
         }
+
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/AddPropertyContentPage.xaml.cs]:
```suggestion
```

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs [hunk 6]
```diff
@@ -1,7 +1,17 @@
...
 {
+    [Android.Runtime.Preserve(AllMembers = true)]
     public static class Constants
     {
+        public const string Verbose = "Verbose";
+        public const string Debug = "Debug";
+        public const string Info = "Info";
+        public const string Warning = "Warning";
+        public const string Error = "Error";
+        public const string BingMapsAuthKey = "paste-key-here";
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs]: Do we have key for our test app?

File: Apps/Contoso.UWP.Demo/MainPage.xaml (hunk 1 above)
Comment (Cloudwu1) [reply Apps/Contoso.UWP.Demo/MainPage.xaml]: Revert it.

File: Apps/Contoso.WPF.Demo.DotNetCore/Property.cs (hunk 2 above)
Comment (Cloudwu1) [reply Apps/Contoso.WPF.Demo.DotNetCore/Property.cs]: Remove it.

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/App.xaml (hunk 4 above)
Comment (Cloudwu1) [reply Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/App.xaml]: Revert and add it to eventfilter toggle.

File: Tests/Microsoft.AppCenter.NET/packages.config [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.NETCore.Platforms" version="1.1.0" targetFramework="net452" />
+  <package id="Microsoft.NETCore.Platforms" version="2.2.2" targetFramework="net452" />
```
Comment (MatkovIvan) [inline Tests/Microsoft.AppCenter.NET/packages.config]: Please revert changes in tests

**Pull request merged**